README.md

```diff
@@ -201,12 +203,12 @@ export default JSONAPIAdapter.extend(AdapterMixin, {
     }
     
     return this._super(...arguments);
-    let url = this._buildURL(modelName, id);
-
-    return urlBuilder(url, requestType, query);
   }
 });
 ```
+requestType - `actionId/path` defined during constructing [modelAction](#Model-actions) or [resourceAction](#Resource-actions)
+returnValue - full URL to action
```

Comment:
`returnValue` - there is not such a value.

If you want to document the output, you can just write: 
```
The function returns a full URL of the action
```
Anyway,
It requires some doc in the code. I can handle that.

Reply:
Changed the info (according to your comment). 
If you think that I should remove it - let me know ;]